flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/EvictingWindowOperator.java

```diff
@@ -231,6 +231,8 @@ public void merge(W mergeResult,
 		// windowAssigner is event time and current timestamp + allowed lateness no less than element timestamp
 		if (isSkippedElement && lateDataOutputTag != null && isElementLate(element)) {
```

Comment:
the if-else conditions are duplicated and inefficient, and can be further combined as

if(isSkippedElement && isElementLate(element)) {
    if(lateDataOutputTag != null) {
        sideOutput(element);
    }
    this.lostDataCount.inc();
}

Reply:
i think it should write in  this way  

```
              if (isSkippedElement && isElementLate(element)) {
			if (lateDataOutputTag != null){
				sideOutput(element);
			} else {
				this.lostDataCount.inc();
			}
		}
```

Reply:
I see. Data being side outputted isn't considered being dropped. 

LGTM